### DISCUSSION TITLE
is it possible to use other ICDAR2023 external dataset?

### DISCUSSION THREAD

**Discussion by IllgamhoDuck** [duchallenge-team/dude#7] [Q&A] [answered]
Hi all!

Is it possible to leverage other ICDAR2023 Datasets?
HierText, SVRD, etc

**Comment by biswassanket** [comment, marked as answer]
Hi @IllgamhoDuck , it is fine to pre-train your model on other external datasets to boost your performance. To leverage more contextual information from external data is most welcome :)

**Reply by jianglong-he-Infrrd** [reply]
Hi @biswassanket, are external datasets also allowed for finetune?

**Reply by pjoziak** [reply]
Anything that can be then explicitly described in submission details is fine. It does not it must be made public, e.g. a description like "Model was pretrained on A, B and C, and fine-tuned on D, E, and F", where any of A-F could look like "A private collection of 2137 invoices aggregated from friends and customers" is still fine.
